st2client/st2client/config_parser.py

```diff
@@ -120,6 +121,8 @@
 
 
 class CLIConfigParser(object):
+    LOG = logging.getLogger(__name__)  # logger instance to use
```

Comment:
Yes, looks like logging messages defined in `CLIConfigParser` didn't work. See previous comment with:
```
No handlers could be found for logger "st2client.config_parser"
```
and @Kami's hint https://github.com/StackStorm/st2/pull/4173#issuecomment-399372838 about passing logger object.